dcos/mesos.py

```diff
@@ -30,14 +30,8 @@ class DCOSClient(object):
 
     def __init__(self):
         config = util.get_config()
-        self._dcos_url = None
-        self._mesos_master_url = None
-
-        mesos_master_url = config.get('core.mesos_master_url')
-        if mesos_master_url is None:
-            self._dcos_url = util.get_config_vals(['core.dcos_url'], config)[0]
-        else:
-            self._mesos_master_url = mesos_master_url
+        self._dcos_url = util.get_config_vals(['core.dcos_url'], config)[0]
```

Comment:
Reading the config everytime seems a little overkill since we are passing the variable along; so either it already threw an exception or it exists. We can be more explicit here with something like:
```
self._dcos_url = config.get("dcos_url")
if self._dcos_url is None:
     raise config.missing_config_exception(['core.dcos_url'])
```